packages/liferay-npm-scripts/contrib/prettier/prettier.js

```diff
@@ -0,0 +1,1080 @@
+						default: 0,
+						range: {
+							start: 0,
+							end: null,
+							step: 1
+						},
+						description:
+							'Format code starting at a given character offset.\nThe range will extend backwards to the start of the first line containing the selected statement.\nThis option cannot be used with --cursor-offset.',
+						pluginDefaults: {}
+					},
+					{
+						name: 'requirePragma',
+						since: '1.7.0',
+						category: 'Special',
+						type: 'boolean',
+						default: false,
+						description:
+							"Require either '@prettier' or '@format' to be present in the file's first docblock comment\nin order for it to be formatted.",
+						pluginDefaults: {}
+					},
+					{
+						name: 'semi',
+						since: '1.0.0',
+						category: 'JavaScript',
+						type: 'boolean',
+						default: true,
+						description: 'Print semicolons.',
+						oppositeDescription:
+							'Do not print semicolons, except at the beginning of lines which may need them.',
+						pluginDefaults: {}
+					},
+					{
+						name: 'singleQuote',
+						since: '0.0.0',
+						category: 'Common',
+						type: 'boolean',
+						default: false,
+						description:
+							'Use single quotes instead of double quotes.',
+						pluginDefaults: {}
+					},
+					{
+						name: 'tabWidth',
+						type: 'int',
+						category: 'Global',
+						default: 2,
+						description: 'Number of spaces per indentation level.',
+						range: {
+							start: 0,
+							end: null,
+							step: 1
+						},
+						pluginDefaults: {}
+					},
+					{
+						name: 'trailingComma',
+						since: '0.0.0',
+						category: 'JavaScript',
+						type: 'choice',
+						default: 'none',
+						description:
+							'Print trailing commas wherever possible when multi-line.',
+						choices: [
+							{
+								value: 'none',
+								description: 'No trailing commas.'
+							},
+							{
+								value: 'es5',
+								description:
+									'Trailing commas where valid in ES5 (objects, arrays, etc.)'
+							},
+							{
+								value: 'all',
+								description:
+									'Trailing commas wherever possible (including function arguments).'
+							}
+						],
+						pluginDefaults: {}
+					},
+					{
+						name: 'useTabs',
+						since: '1.0.0',
+						category: 'Global',
+						type: 'boolean',
+						default: false,
+						description: 'Indent with tabs instead of spaces.',
+						pluginDefaults: {}
+					},
+					{
+						name: 'vueIndentScriptAndStyle',
+						since: '1.19.0',
+						category: 'HTML',
+						type: 'boolean',
+						default: false,
+						description:
+							'Indent script and style tags in Vue files.',
+						pluginDefaults: {}
+					}
+				]
+			};
+			/* eslint-enable sort-keys */
+		}
+	},
+
+	resolveConfig(filepath, options = {}) {
+		const prettier = prepare(filepath);
+
+		if (prettier) {
+			return prettier.resolveConfig(filepath, options);
+		} else {
+			return Promise.resolve(null);
+		}
+	},
+
+	version: '1.19.1'
+};
+
+const PORTAL_WEB = 'portal-web';
```

Comment:
I'll inline this. It's only here in a constant because during an earlier iteration of this code I was using it in two places.